### DISCUSSION TITLE
To label the cell type, should I use reference mapping if I have 3 different groups(normal. Vehicle, drug treatment) in my query dataset while 3 different groups (normal, disease, treatment) in my reference dataset?

### DISCUSSION THREAD

**Discussion by lkqnaruto** [satijalab/seurat#4510] [Q&A]
HI

I was trying to do the cell type annotation for my dataset, which contains 3 different groups: Normal(4 samples), Vehicle treatment group(8 samples), drug treatment(8 samples). My goal is to give cell type annotation for EACH group. But my reference dataset contains cell type information but for 3 different groups (Normal, Disease, another different Drug treatment).

I have some questions that bothers me for a while, and I really appreciate if I can have your feedback.
 
Q1: I was wondering whether reference mapping would be appropriate for my goal? My potential workflow would be integrate the query dataset and the reference dataset together following the[ reference mapping vignettes](https://satijalab.org/seurat/articles/integration_mapping.html). By integration of query data and reference data, I would have my query dataset easily annotated.

Q2: Should I ONLY use Normal reference to infer the normal query dataset? If so, what should I do for the rest? Since the other 2 groups(Vehicle treatment, drug treatment) in query dataset are completely different with the 2 groups(Disease, drug treatment) in reference dataset

Q3: I notice that reference mapping is a supervised analysis guided by a reference dataset, which, by integration, maps both query dataset and inference dataset to a common space so that cells groups together would have same cell type(correct me if I am wrong). I wonder what is the main difference comparing to some other toolkit like SIngleR and ClustifyR. etc? I know SingleR infer the cell type label based on the Spearman's correlation score between query data and inference data.

Thank you.

**Comment by timoast** [comment]
We have found reference mapping of disease or treated datasets to a healthy reference generally works very well. If you had an extreme case where there was an entirely new cell type specific to your disease condition, you may want to try an unsupervised clustering as well, but this seems to be rare in practice.
